src/components/vive-controls.js

```diff
@@ -66,9 +67,34 @@ module.exports.Component = registerComponent('vive-controls', {
     var data = this.data;
     var objUrl = 'url(' + VIVE_CONTROLLER_MODEL_OBJ_URL + ')';
     var mtlUrl = 'url(' + VIVE_CONTROLLER_MODEL_OBJ_MTL + ')';
+
     // handId: 0 - right, 1 - left
     var controller = data.hand === 'right' ? 0 : 1;
-    el.setAttribute('tracked-controls', 'controller', controller);
+
+    // interrogate gamepads ourselves to see what we need to specify
+    // (to do a priori, we need to figure out whether to use Vive or Rift)
+    var gpads = navigator.getGamepads && navigator.getGamepads();
+    if (typeof gpads !== 'undefined') {
+      for (var i = 0; i < gpads.length; i++) {
+        if (gpads[i].id.indexOf('Oculus Touch') === 0) {
```

Comment:
the original intent was to make hand-controls left and right hands just work whether Oculus Touch or Vive wands.  internally, hand-controls uses vive-controls, which uses tracked-controls.

maybe what is really needed is to refactor vive-controls into modeled-tracked-controls, and then vive-controls, hand-controls and oculus-touch-controls could all leverage that -- but those are much larger changes (and maybe not backward compatible) so thought getting things functional first might be wise.

if what you are saying is that tracked-controls should be doing what vive-controls / modeled-tracked-controls does, that could work as well, but I assumed there was an explicit decision to keep tracked-controls more minimal.  (may want to remove the default to OpenVR Gamepad in there, since the code will already filter out gamepads without poses)